### PR TITLE
ent-go: init 0.10.0

### PR DESCRIPTION
**Pull request by superherointj**
ent-go: init 0.10.0

### CODE REVIEW

File: pkgs/development/tools/ent/default.nix [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  src = fetchFromGitHub {
+    owner = "ent";
+    repo = "ent";
+    rev = "v${version}";
+    sha256 = "sha256-TG08GRo1gNxC5iHt/Md5WVWaEQ1m2mUDGqpuxw8Pavg=";
+  };
+
+  vendorSha256 = "sha256-dK1C59Euj15nyHber19qop0Vr5R/u8Ne6S7sN25HQ9E=";
+
+  subPackages = [ "cmd/ent/" ];
```
Comment (zowoq) [inline pkgs/development/tools/ent/default.nix:16]:
```suggestion
  subPackages = [ "cmd/ent" ];
```

File: pkgs/development/tools/ent/default.nix [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+  ldflags = [
+    "-s"
+    "-w"
+  ];
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  doCheck = true;
+
```
Comment (zowoq) [inline pkgs/development/tools/ent/default.nix:26]:
```suggestion
```

Comment (superherointj) [reply pkgs/development/tools/ent/default.nix]: Done

File: pkgs/development/tools/ent/default.nix (hunk 1 above)
Comment (superherointj) [reply pkgs/development/tools/ent/default.nix]: Done

**Comment by zowoq** [comment]
@ofborg eval

**Pull request merged**